src/components/toast/Toast.tsx

```diff
@@ -81,7 +82,7 @@ Toast.propTypes = {
   className: PropTypes.string,
   closeIcon: PropTypes.bool,
   content: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
-  leftIcon: PropTypes.string,
+  leftIcon: PropTypes.any,
```

Comment:
Same comment as above.
I suggest to use ` PropTypes.string as PropTypes.Validator<TkIcon>,`

Reply:
Oh very nice!, thaanks :)